infra/examples-dev/aws-all/variables.tf

```diff
@@ -186,6 +186,11 @@ variable "custom_api_connector_rules" {
   default     = {}
 }
 
+variable "use_api_gateway" {
```

Comment:
Rename to `use_api_gateway_v2` to make it clear that is for v2 ?

Reply:
🤔  feels hacky to me to expose version numbers, but OK

Reply:
Agree, it was a suggestion, but on the other hand we had `use_api_gateway` and then in the description we enforce that is *enable api gateway v2* (as opposite to use v1), so yeah, agree that is weird to expose version numbers...